### PR TITLE
Bump CoreOS AMI to v1745.7.0

### PR DESCRIPTION
**Pull request by zen-dog**
## High-level description
Bump CoreOS AMI to v1745.7.0 allowing for 5th generation of EC2 instances. The AMIs are taking from the CoreOS [documentation page](https://coreos.com/os/docs/latest/booting-on-ec2.html)

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS-20093](https://jira.mesosphere.com/browse/DCOS-20093) 
  - [DCOS_OSS-381](https://jira.mesosphere.com/browse/DCOS_OSS-381)
  - [DCOS-38403](https://jira.mesosphere.com/browse/DCOS-38403)

## Checklist for all PRs

  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]
___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by mesosphere-mergebot-dev** [comment]
This repo has @mesosphere-mergebot integration. You can interact with the following commands.
```
@mesosphere-mergebot bump-ee  
@mesosphere-mergebot merge-it  
@mesosphere-mergebot label [Request For Comment |Work In Progress |Holding |Ship It |Ready For Review] 
@mesosphere-mergebot override-status  
```

* PR creators can apply one of  `[Ready For Review |Work In Progress]`. [Owners](https://github.com/dcos/dcos/blob/master/owners.json) can apply any label.

**Comment by zen-dog** [comment]
@mesosphere-mergebot bump-ee

File: gen/build_deploy/aws.py [hunk 1]
```diff
@@ -128,15 +128,15 @@ def validate_provider(provider):
...
+        'coreos': 'ami-ab20efd4',
+        'stable': 'ami-ab20efd4',
         'el7': 'ami-965345f8',
         'el7prereq': 'ami-f1aee697',
         'natami': 'ami-55c29e54'
     },
     'ap-southeast-1': {
-        'coreos': 'ami-aacc7dc9',
-        'stable': 'ami-aacc7dc9',
+        'coreos': 'ami-961ac5f4',
```
Comment (gauripowale) [inline gen/build_deploy/aws.py]: This AMI is for `ap-southeast-2`, the `ap-southeast-1` should be `ami-3cded940`.
I think you accidentally skipped updating `ap-southeast-2` (line 144), which should be updated to `ami-961ac5f4`

Comment (zen-dog) [reply gen/build_deploy/aws.py]: You're absolutely right! Good catch. Fixed.

**Comment by orsenthil** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/aws/cloudformation/simple https://jira.mesosphere.com/browse/DCOS_OSS-2115

**Comment by gauripowale** [comment]
@mesosphere-mergebot bump-ee

**Comment by mesosphere-mergebot-dev** [comment]
Your pull request's branch is not based on the most recent version of `master`. Please rebase your changes against this repo's `master` branch.

**Comment by zen-dog** [comment]
@mesosphere-mergebot bump-ee

**Comment by mesosphere-mergebot-dev** [comment]
@zen-dog, an [owner](https://raw.githubusercontent.com/dcos/dcos/master/owners.json) other than the PR creator needs to issue that command.

**Comment by mesosphere-mergebot-dev** [comment]
@zen-dog, an [owner](https://raw.githubusercontent.com/dcos/dcos/master/owners.json) other than the PR creator needs to issue that command.

**Comment by jgehrcke** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/aws/cloudformation/simple https://jira.mesosphere.com/browse/DCOS_OSS-2115

**Comment by orsenthil** [comment]
Hi @zen-dog  - I think, your `bump-ee` after the final change was not successful. You might have do the bump-ee again after updating your branch will changes from master. 

- Apologies. This was not your mistake, and it was not obvious. The bump-ee command had not registered and did not give you response by updating the EE PR. 
- Trying bump-ee again should be okay, you will have to update your branch against the master because master has moved. ([more info on why](https://github.com/mesosphere/dcos-enterprise/blob/master/docs/workflow.md), etc)

Please update this branch with changes from master, and do a bump-ee again, I will approve both the open and EE PRs again. Thank you!

**Comment by zen-dog** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
Hey @zen-dog  - It looks like the networking failure in the CloudFormation Testsuite is directly related to the changes from this. - Please look at these tests [consistently failing on this PR](https://teamcity.mesosphere.io/viewType.html?buildTypeId=DcOs_Open_Test_IntegrationTest_AwsCloudFormationSimple&branch_DcOs_Open_Test_IntegrationTest=pull%2F2992&tab=buildTypeStatusDiv) when the [master branch is stable](https://teamcity.mesosphere.io/viewType.html?buildTypeId=DcOs_Open_Test_IntegrationTest_AwsCloudFormationSimple&tab=buildTypeStatusDiv&branch_DcOs_Open_Test_IntegrationTest=%3Cdefault%3E) - We should not be merging this PR without the corresponding networking fix.  

The[ previous override](https://github.com/dcos/dcos/pull/2992#issuecomment-400778979) I did was a mistake. We could have avoided the unfortunate mechanical churn if we had identified the real issue that observed since the origin of this PR.

**Comment by zen-dog** [comment]
> Update to CoreOS v1745.7.0 is leading to networking failures in DC/OS

Isn't that the [usual flakiness](https://jira.mesosphere.com/browse/DCOS_OSS-2115) of the networking tests?

**Comment by mesosphere-mergebot-dev** [comment]
:wave: This PR has been inactive for 3 days. Please review the status checks to see what needs to be done to move it forward or change the label to Work in Progress. Thank you!

**Comment by orsenthil** [comment]
Hi Aleksey - Nope, it isn't the usual flakiness. There is networking issue and it is assigned to the networking team at the moment - https://jira.mesosphere.com/browse/DCOS_OSS-3707

**Comment by orsenthil** [comment]
@mesosphere-mergebot label work in progress

**Pull request closed without merging**